src/main/scala/org/broadinstitute/dsde/workbench/sam/api/TermsOfServiceRoutes.scala

```diff
@@ -79,15 +93,20 @@ trait TermsOfServiceRoutes {
           } ~
           // The {user_id} route must be last otherwise it will try to parse the other routes incorrectly as user id's
           pathPrefix(Segment) { userId => // api/termsOfService/v1/user/{userId}
-            pathEndOrSingleSlash {
-              get {
-                complete(StatusCodes.NotImplemented)
-              }
-            } ~
-            pathPrefix("history") { // api/termsOfService/v1/user/{userId}/history
+            validate(samUserIdPattern.matches(userId), "User ID must be alpha numeric") {
```

Comment:
Do we need this check?

Reply:
Greg added it, idk that we NEED it but I think extra validation and user feedback is usually a good thing 🤷